include/RAJA/exec-cuda/forall_cuda.hxx

```diff
@@ -176,7 +182,7 @@ RAJA_INLINE void forall(cuda_exec<BLOCK_SIZE>,
 
   RAJA_FT_BEGIN;
 
-  forall_cuda_kernel<<<gridSize, BLOCK_SIZE>>>(loop_body, begin, len);
```

Comment:
Yes,  for generality we could add some logic to check which reduction variables are in scope based on ID and also it's type size to compute it's offset, or do something easier since reduction variables are the only thing using shared memory, and use an array based on max reduce vars .. what do you think?